src/metaswitch/clearwater/cluster_manager/pdlogs.py

```diff
@@ -0,0 +1,83 @@
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version, along with the "Special Exception" for use of
+# the program along with SSL, set forth below. This program is distributed
+# in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details. You should have received a copy of the GNU General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# The author can be reached by email at clearwater@metaswitch.com or by
+# post at Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+#
+# Special Exception
+# Metaswitch Networks Ltd  grants you permission to copy, modify,
+# propagate, and distribute a work formed by combining OpenSSL with The
+# Software, or a work derivative of such a combination, even if such
+# copying, modification, propagation, or distribution would otherwise
+# violate the terms of the GPL. You must comply with the GPL in all
+# respects for all of the code used other than OpenSSL.
+# "OpenSSL" means OpenSSL toolkit software distributed by the OpenSSL
+# Project and licensed under the OpenSSL Licenses, or a work based on such
+# software and licensed under the OpenSSL Licenses.
+# "OpenSSL Licenses" means the OpenSSL License and Original SSLeay License
+# under which the OpenSSL Project distributes the OpenSSL toolkit software,
+# as those licenses appear in the file LICENSE-OPENSSL.
+
+from metaswitch.common.pdlogs import PDLog
+
+STARTUP = PDLog(number=PDLog.CL_CLUSTER_MGR_ID+1,
+                desc="clearwater-cluster-manager has started.",
+                cause="The application is starting.",
+                effect="Normal.",
+                action="None.",
+                priority=PDLog.LOG_NOTICE)
+EXITING = PDLog(number=PDLog.CL_CLUSTER_MGR_ID+2,
+                desc="clearwater-cluster-manager is exiting.",
+                cause="The application is exiting.",
+                effect="Datastore cluster management services are no longer available.",
+                action="This occurs normally when the application is stopped. Wait for monit to restart the application.",
+                priority=PDLog.LOG_ERR)
+EXITING_BAD_CONFIG = PDLog(number=PDLog.CL_CLUSTER_MGR_ID+3,
+                           desc="clearwater-cluster-manager is exiting due to bad configuration.",
+                           cause="clearwater-cluster-manager was started with incorrect configuration.",
+                           effect="Datastore cluster management services are no longer available.",
+                           action="Verify that the configuration files in /etc/clearwater/ are correct according to the documentation. In particular, ensure that local_ip, management_local_ip, log_level, local_site_name and remote_site_name are set correctly.",
```

Comment:
I'm not sure about listing the options here. It feels like we'll never remember to update these.